server/src/testFixtures/java/io/crate/testing/TestingHelpers.java

```diff
@@ -183,6 +183,13 @@ public static NodeContext createNodeContext(User user, AbstractModule... additio
         );
     }
 
+    public static NodeContext createNodeContext(List<User> users, AbstractModule... additionalModules) {
```

Comment:
Looks like the existing overload `createNodeContext(User user, AbstractModule... additionalModules)` is now only used inside this class. Can we remove it/replace with your new one?

Reply:
Nope, it has 32 usages overall.

Reply:
Strange, if I check for usages, createNodeContext(User user, AbstractModule... additionalModules) only has 2 usages here https://github.com/crate/crate/blob/master/server/src/testFixtures/java/io/crate/testing/TestingHelpers.java#L176 and here https://github.com/crate/crate/blob/master/server/src/testFixtures/java/io/crate/testing/SqlExpressions.java#L77.
Do you maybe mix it up with usages of createNodeContext(AbstractModule... additionalModules) ?

Reply:
You're right, I mixed up with imports, of the same name, but the other signature, removing, thank you!